hw/ip/sram_ctrl/data/sram_ctrl.hjson

```diff
@@ -126,6 +152,33 @@
         }
       ]
     }
+    { name: "EXEC_REGWEN",
+      desc: "Lock register for execution enable register.",
+      swaccess: "rw0c",
+      hwaccess: "none",
+      fields: [
+        { bits: "0"
+          desc: '''
+                When cleared to zero, !!CTRL can not be written anymore.
+                ''',
+          resval: 1
+        }
+      ]
+    }
+    { name: "EXEC",
+      desc: "Sram execution enable.",
+      swaccess: "rw",
+      hwaccess: "hro",
+      regwen:   "EXEC_REGWEN"
+      fields: [
+        { bits: "2:0",
+          name: "EN",
+          desc: '''
+                  Write 5 (3b101) to this field to enable execution from sram
```

Comment:
nit: sram -> SRAM for consistency within this file

Reply:
As for the value, I believe this needs to be `0xAA (8b10101010)` since the CSR value is connected straight through when enabled via the fuse.

Reply:
o yeah that's right, forgot about the 8b min

Reply:
o actually this is meant to be 3b. The tlul definition for enable / disable is only that wide.
I explicitly disconnected it from the otp value since i didn't want to propagate that everywhere.

Reply:
ah ok I see. never mind then.